### PR TITLE
Add groups mode on progress page

### LINKED ISSUE #970: Selecting group polygon on progress map zooms and shows filtered blockfaces with action bar

### PR DESCRIPTION
**Pull request by maurizi**
When a group polygon is clicked, loads the group's blockfaces and loads
a popup with group info into the action bar

Fixes #970

### CODE REVIEW

File: src/nyc_trees/apps/survey/helpers.py [hunk 1]
```diff
@@ -35,3 +37,20 @@ def _teammate_user_context(user):
...
+    group_blocks = Territory.objects \
+        .filter(group=group) \
+        .values_list('blockface_id', flat=True)
+
+    group_blocks_count = group_blocks.count()
+
+    if group_blocks_count > 0:
+        completed_blocks = Survey.objects \
+            .filter(blockface_id__in=group_blocks) \
+            .distinct('blockface')
```
Comment (RickMohr) [inline src/nyc_trees/apps/survey/helpers.py]: We shouldn't count a survey whose blockface has `is_available == True`

Comment (RickMohr) [inline src/nyc_trees/apps/survey/helpers.py]: Also, we should probably include only active groups.

File: src/nyc_trees/apps/survey/views.py [hunk 2]
```diff
@@ -163,6 +167,42 @@ def get_formatted_expiration_date(reservation):
     ]
 
 
+def group_borders_geojson(request):
+    groups = Group.objects.all() \
+        .prefetch_related(
```
Comment (RickMohr) [inline src/nyc_trees/apps/survey/views.py]: We should probably include only active groups.

Comment (maurizi) [inline src/nyc_trees/apps/survey/views.py]: Agreed

**Comment by maurizi** [comment]
@RickMohr I've fixed both of those issues

**Comment by RickMohr** [comment]
Changes look good. I'll wait to try it with multi-polygons.

**Comment by maurizi** [comment]
@RickMohr I've modified `border` to be a `MultiPolygon`

**Comment by maurizi** [comment]
I'm working on fixing unit tests now

**Comment by RickMohr** [comment]
+1

**Pull request merged**